Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -501,7 +506,14 @@ class StatusTableViewController: UITableViewController, UIGestureRecognizerDeleg
                     let numberValue = NSNumber(double: iob).descriptionWithLocale(locale)
                     cell.detailTextLabel?.text = "\(numberValue) Units"
                 } else {
-                    cell.detailTextLabel?.text = emptyValueString
+                    //x22 BIoB Value from IOB Chart Decay Data
+                    dataManager.doseStore.getInsulinOnBoardValues(startDate: NSDate(), endDate: NSDate().dateByAddingTimeInterval(300)) { (values, error) -> Void in
+                        if let error = error {
+                            self.dataManager.logger.addError(error, fromSource: "DoseStore")
+                        }else{
+                            cell.detailTextLabel?.text = "\(NSString(format: "%.2f", values[0].value)) Units"
```

Comment:
There's a few things to note here. First, formatting numbers to strings should be done by customizing an instance of `NSNumberFormatter`. This will ensure the user's locale is respected; for example, when choosing a decimal separator: `7.3` vs `7,3`.

Second, the pattern `"%@ Units"` should allow for internationalization, by wrapping it in the `NSLocalizedString` function. If someone does want to translate Loop, they'll be able to do that easily if all english labels are properly setup ahead of time.

You might notice there's lots of cells in this section that aren't using `NSLocalizedString`... and that's because I've planned to continue phasing this entire text-based section of the table view out of the UI.